projects/storefrontlib/shared/components/card/card.component.ts

```diff
@@ -79,11 +96,22 @@ export class CardComponent implements OnInit {
     this.editMode = true;
   }
 
+  setSaveMode(): void {
+    this.saveMode = true;
+  }
+
   cancelEdit(): void {
     this.editMode = false;
     this.cancelCard.emit(5);
   }
 
+  cancelSave(): void {
+    this.saveMode = false;
+    const saveCardEvent: SaveCardEvent = { saveMode: this.saveMode };
+    this.saveCard.emit(saveCardEvent);
+    this.cancelCard.emit(5);
```

Comment:
what is meant by cancelSave and why does it emit both savedCard and cancelCard event?
i would expect that you can either cancel or save your changes, but not both.